renv/core.py

```diff
@@ -257,13 +262,15 @@ def create_configuration(self, context):
             config_dict["R_ENV_HOME"] = context.env_R_home
             config_dict["R_ABS_HOME"] = context.abs_R_home
             config_dict["R_INCLUDE_DIR"] = context.env_R_include
+            config_dict["R_DOC_DIR"] = context.env_R_doc
+            config_dict["R_SHARE_DIR"] = context.env_R_share
             config_dict["R_VERSION"] = context.R_version
             # Package lists
             config_dict.update(__DEFAULT_CONFIG__)
             pkg_lists = self.format_pkg_list(config_dict)
             config_dict.update(pkg_lists)
             config_dict.update(user_config)
-            logging.info(f"Config Dictionary:  {config_dict}")
+            logging.info("Config Dictionary:  %s" % config_dict)
 
             # Dump the configuration dictionary to the YAML file in the R environment HOME
             yaml.dump(config_dict, f, default_flow_style=False)
```

Comment:
Could we remove all the commented-out code and instead open an issue and refer to them by issue number, just like what you have with ` See issue 2487` ?

Reply:
Sure that was actually copied from python's venv module.  But that would probably be a good practice.  For now, I'm going to leave it as is.  My next PR will be a fairly extensive rework in order to break the functions up, and to handle the OS logic in separate classes (similar to what I just did in rinse; e.g. BaseInstallR, LinuxInstallR, MacInstallR, WindowsInstallR).  I envision something like BaseRenvBuilder, LinuxRenvBuilder, MacRenvBuilder, and WIndowsRenvBuilder.  These comments will be removed and appropriate issues will be created, if they are needed.